rpcs3/rpcs3qt/welcome_dialog.h

```diff
@@ -0,0 +1,14 @@
+#ifndef WELCOME_DIALOG_H
+#define WELCOME_DIALOG_H
+
+#include <QDialog>
+
+class welcome_dialog : public QDialog
+{
+	Q_OBJECT
+
+public:
+	explicit welcome_dialog(QWidget* parent = nullptr);
+};
```

Comment:
Add those here if you want to prevent users from escaping the dialog without checking stuff
```
private:
	void keyPressEvent(QKeyEvent* event)
	{
		// this won't work with Alt+F4, the window still closes
		if (m_disable_cancel && event->key() == Qt::Key_Escape)
		{
			event->ignore();
		}
	}
	void closeEvent(QCloseEvent* event)
	{
		// spontaneous: don't close on external system level events like Alt+F4
		if (m_disable_cancel && event->spontaneous())
		{
			event->ignore();
		}
	}
```